types/types.go

```diff
@@ -156,6 +157,7 @@ type ImageDestination interface {
 	// PutBlob writes contents of stream and returns data representing the result (with all data filled in).
```

Comment:
The “all data filled in” is not currently true for return values of `PutBlob`; most (all?) implementations currently fill in only digest and size.

Same for `ReapplyBlob`.

(It _would_ be nice if it were true eventually.)